test/integration/controlplane/audit_test.go

```diff
@@ -392,6 +444,46 @@ func runTestWithVersion(t *testing.T, version string) {
 			testAuditCrossGroupSubResource(t, version, tc.expEvents, tc.namespace, kubeclient, logFile)
 		})
 	}
+
+	for _, tc := range rawHttpRequestGroupTestCases {
+		t.Run(fmt.Sprintf("raw-http-request-%s.%s.%s", version, tc.auditLevel, tc.namespace), func(t *testing.T) {
+			testAuditRawHttpRequest(t, version, tc.expEvents, tc.namespace, kubeclient, logFile)
+		})
+	}
+}
+
+func testAuditRawHttpRequest(t *testing.T, version string, expEvents []utils.AuditEvent, namespace string, kubeclient *clientset.Clientset, logFile *os.File) {
+	var lastMissingReport string
+	sendHttpRequest(t, kubeclient, "GET", fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace))
+	sendHttpRequest(t, kubeclient, "OPTIONS", fmt.Sprintf("/api/v1/namespaces/%s/configmaps/audit-configmap", namespace))
+
+	if err := wait.Poll(500*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
+		// check for corresponding audit logs
+		stream, err := os.Open(logFile.Name())
+		if err != nil {
+			return false, fmt.Errorf("unexpected error: %v", err)
+		}
+		defer stream.Close()
+		missingReport, err := utils.CheckAuditLines(stream, expEvents, versions[version])
+		if err != nil {
+			return false, fmt.Errorf("unexpected error: %v", err)
+		}
+		if len(missingReport.MissingEvents) > 0 {
+			lastMissingReport = missingReport.String()
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		t.Fatalf("failed to get expected events -- missingReport: %s, error: %v", lastMissingReport, err)
+	}
+}
+
+func sendHttpRequest(t *testing.T, kubeclient *clientset.Clientset, httpVerb string, url string) {
+	// We don't care about emitter errors here. However, if the tests get broken, you might want to check them out.
+	err := kubeclient.RESTClient().Verb(httpVerb).Timeout(32 * time.Second).RequestURI(url).Do(context.TODO()).Error()
```

Comment:
```suggestion
	err := kubeclient.RESTClient().Verb(httpVerb).Timeout(wait.ForeverTestTimeout).RequestURI(url).Do(context.TODO()).Error()
```

Reply:
just for consistency

Reply:
I see , this is the same timeout that shows up in the url, scratch my comment

Reply:
I noticed that the API Server adds a default timeout equal 32s (even if it's not present in my URL). In order to make it a bit more durable, I decided to force 32s on the HTTP Client side and expect the same value in the Audit logs. This way, it will work even if the default API Server timeout gets increased/decreased.